hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/FhirPayloadExtractor.java

```diff
@@ -1,55 +1,33 @@
 package ca.bc.gov.hlth.hnsecure;
 
-import net.minidev.json.JSONArray;
-import net.minidev.json.JSONObject;
-import net.minidev.json.parser.JSONParser;
+import java.io.UnsupportedEncodingException;
+
 import org.apache.camel.Handler;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.UnsupportedEncodingException;
-import java.nio.charset.StandardCharsets;
-import java.util.Base64;
+import ca.bc.gov.hlth.hnclientv2.Util;
+import ca.bc.gov.hlth.hnclientv2.json.FHIRJsonMessage;
+import ca.bc.gov.hlth.hnclientv2.json.FHIRJsonUtil;
```

Comment:
HNClient is a separate project (and deployed separately) we can't share dependencies between these projects. 
Can you add this functionality as classes to the HNI-ESB project? 

(Maybe in the future we could look at creating a library we can share as a dependency for both)

Reply:
Agreed, I think we should consider splitting/sharing this component as part of a reference implementation. but that's something we can look into later.

Reply:
Thanks guys,
I will add this functionality into the HNI-ESB project and let you know.

Tony

Reply:
@tschia 
Hi Trevor, this functionality has been added into the HNI-ESB project and please review the code.

Thanks,
Tony